js/securesubmit/checkout-form.js

```diff
@@ -857,13 +856,19 @@ document.observe('dom:loaded', function () {
                     }
                 });
 
-                if (window.amastyCompleteCheckoutOriginal && document.querySelector('#amscheckout-onepage')) {
-
-                    document.querySelector('#ss-banner').style.backgroundSize = '325px 40px';
-                    document.querySelector('#cc-number').className = 'amasty_one_page_checkout';
-                    document.querySelector('#expiration-date').className = 'amasty_one_page_checkout';
-                    document.querySelector('#ccv').className = 'amasty_one_page_checkout';
-
+                if (window.secureSubmitAmastyCompleteCheckoutOriginal && document.getElementById('amscheckout-onepage')) {
+                    if(document.getElementById('ss-banner'){
+                        document.getElementById('ss-banner').style.backgroundSize = '325px 40px';
+                    }
+                    if(document.getElementById('cc-number')){
+                        document.getElementById('cc-number').className = 'securesubmit_amasty_one_page_checkout';
+                    }
+                    if(document.getElementById('expiration-date')){
+                        document.getElementById('expiration-date').className = 'securesubmit_amasty_one_page_checkout';
+                    }
+                    if(document.getElementById('ccv')){
+                        document.getElementById('ccv').className = 'securesubmit_amasty_one_page_checkout';
+                    }
```

Comment:
@ttnnkkrr These calls to `document.getElementById` should be abstracted to a single call. This removes duplication as well as the multiple queries against the DOM.